### ISSUE TITLE
All - Dynamically determine the Javascript include URL

### ISSUE THREAD

**Issue by bitingmidge** [bezlio/bezlio-apps#213]
Add code to all apps to dynamically determine the correct javascript include path at runtime to remove need to edit paths when pushing from development to master branch. The prototype is already completed in the CRM Calls app.

Add a setConfig() function with code similar to the following:
```
bezl.vars.config = {};

bezl.vars.config.baseUrl = "";
// If the baseURL is not set in the line above then we will determine a sane default based on the URL
if (bezl.vars.config.baseUrl == "") {
    var tld = "";
    if (window.location.href.indexOf("portal-dev.bezl.io") !== -1) {
        tld = "https://bezlio-apps-dev.bezl.io";
    } else {
        tld = "https://bezlio-apps.bezl.io";
    }
    bezl.vars.config.baseUrl = tld + "/roles/sales-rep/crm-calls-view/js/";
}
```
Add the following to the top of onDataChange():
`bezl.functions['setConfig']();`

**NOTE:** _when issue bezlio/bezlio#360 is resolved it can be moved to ngOnInit() where it really belongs._

Anywhere you need to specify the path to a javascript include use something similar to the following:
```
require([bezl.vars.config.baseUrl + 'onDataChange.js'], function(functions) {
	functions.onDataChange(bezl)
});
```

**Comment by bitingmidge** [comment]
Closed with pull request #224

**Comment by bitingmidge** [comment]
Need to rework this a bit. There are references to /libraries/epicor etc. that need to be based off of a more generic root path than what I currently have.
